dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageAPI.java

```diff
@@ -237,6 +241,16 @@
      */
     public Language getFallbackLanguage(final String languageCode);
 
-
+	/**
+	 * Return all the languages for a specific contentletInode
+	 *
+	 * @param contentletInode
+	 * @param user
+	 * @return
+	 * @throws DotSecurityException if the user dont have read permission in the contentlet
+	 * @throws DotDataException
+	 */
+	public List<Language> getAvailableContentPageLanguages(final String contentletInode, final User user)
```

Comment:
As this method works with any type of content we could change the name, is not exclusive of HTMLPages